parcelable/src/androidTest/kotlin/com/arkivanov/essenty/parcelable/AndroidParcelableContainerRobolectricTest.kt

```diff
@@ -23,6 +24,19 @@ class AndroidParcelableContainerRobolectricTest {
         assertEquals(value, consumedValue)
     }
 
+    @Test
+    fun WHEN_parcelling_ignored_THEN_returns_default_value() {
```

Comment:
This class is for `ParcelableContainer` tests. It looks like `IgnoredOnParcel` could be tested separately in its own `IgnoredOnParcelTest` class. Just save the `Ignored` object into `Parcel`, and then restore it.